docs/docs/providers/legacy/client_context_aware_configuration.md

```
@@ -239,6 +267,12 @@ Create a new client in the Client Factory
 createCacClient:: Tenant -> Interval -> Hostname -> IO (Either Error ())
 ```
 
+To configure cache properties explicitly:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)